docs/contributing/release.md

```diff
@@ -0,0 +1,19 @@
+# How to release a new version of Claudie
+
+Release process of Claudie consists of a few manual steps and a few automated steps.
+
+## Manual steps
+
+Whoever is responsible for creating a new release has to:
+
+1. Write a new entry to a relevant [Changelog document](https://github.com/Berops/claudie/tree/master/docs/CHANGELOG)
+2. Write a release notes on the Release page
+3. Publish a release
+
+## Automated steps
+
+After a new release is published, a [release pipeline](https://github.com/Berops/claudie/blob/master/.github/workflows/release.yml) will run which will:
+
+1. Build a new images with a release tag
+2. Push them to the container registry where anyone can pull them
+3. Append Claudie manifest files to the release, containing reference to the images from a release
```

Comment:
not nitpicking here, this is simply unclear to me as I read it.
do we mean here that the manifest files are published alongside links to the GHCR-stationed images, or even something else?
could you, please, explain? @MiroslavRepka

Reply:
The way Claudie is deployed is by manifests in `manifests/claudie`. These manifests contain everything needed for a successful deployment.

The "links to the images" are included in those manifests (see [`kustomization.yaml`](https://github.com/Berops/claudie/blob/master/manifests/claudie/kustomization.yaml)), therefore, there is no need to add anything else to the release, just the manifests suffice.

Reply:
right, thanks for the explanation.
that wording is still confusing to me though (again, not trying to nitpick).

could that perhaps be changed, like so:
```diff
-3. Append Claudie manifest files to the release, containing reference to the images from a release
+3. Append Claudie manifest files (containing the just-released images' references) to the release
```